docs/design-docs/01 - mvp/01-mvp-design.es.md

```diff
@@ -28,8 +35,20 @@ En la primera versión, vamos a priorizar las siguientes características:
 
 ---
 
+## 🤝 Decisiones clave
+
+1. Arquitectura en capas y modular dentro de un único módulo:
+    - Nos permitirá avanzar más rápido
+    - Se puede refactorizar a multimódulo una vez que el proyecto escale
+2. Patrón de repositorio para la capa de datos:
+    - Nos permite establecer una adecuada separación de responsabilidades
+    - Simplifica las pruebas
+    - Práctica estándar en la industria
+3. Flujo de datos unidireccional en la capa de presentación
```

Comment:
If the decisions were more complicated, we should have included:
- Every approach that was considered
- Pro & con of each approach
- Which approach we took and why (how did we manage the tradeoffs)

In this case tho, i'm not adding too much details because some of the decisions are rather straightforward.

Reply:
@Bruno125 I agree! but I'd add one more key topic, the backend

Reply:
and I'd also suggest setting as TBD and creating a new ticket to define that topic.

Reply:
I think that Backend implementation (Firebase, Kotlin, whatever) will need its own documentation. Maybe adding just a disclaimer in order to mention it? @Bruno125 @pedrorau

Reply:
@jflavio11 I agree!

Reply:
Disclaimer added in English and Spanish docs.

Merging